extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -1922,6 +1961,33 @@ export class AmpDatePicker extends AMP.BaseElement {
   }
 }
 
+const AMP_READONLY_DATA_ATTR = 'iAmphtmlReadonly';
+
+const AMP_DATE_BLUR_DATA_ATTR = 'iAmphtmlDateBlur';
+
+/**
+ *
+ * @param {!Element} element
+ * @param {boolean} toggle
+ */
+function setNonValidationReadonly(element, toggle) {
+  if (!toggle) {
+    delete element.dataset[AMP_READONLY_DATA_ATTR];
+    return;
+  }
+
+  element.dataset[AMP_READONLY_DATA_ATTR] = true;
```

Comment:
Would add a comment that says the actual `readonly` is set when the element gets focus.

Reply:
Added comment to the method body